src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCourtLocalAuthoritiesEndpointTest.java

```diff
@@ -0,0 +1,163 @@
+package uk.gov.hmcts.dts.fact.admin;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.dts.fact.model.admin.LocalAuthority;
+import uk.gov.hmcts.dts.fact.util.AdminFunctionalTestBase;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.HttpStatus.*;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.*;
+
+
+@ExtendWith(SpringExtension.class)
+public class AdminCourtLocalAuthoritiesEndpointTest extends AdminFunctionalTestBase {
+
+
+    private static final String ADMIN_COURTS_ENDPOINT = "/admin/courts/";
+    private static final String LOCAL_AUTHORITIES_PATH = "localAuthorities";
+    private static final String ALL_LOCAL_AUTHORITIES_FULL_PATH = ADMIN_COURTS_ENDPOINT + LOCAL_AUTHORITIES_PATH;
+    private static final String AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_SLUG = "aylesbury-county-court-and-family-court/";
+    private static final String AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_AREAS_OF_LAW = "Money claims/";
+
+    private static final String AYLESBURY_COURT_LOCAL_AUTHORITIES_AREAS_OF_LAW_PATH = ADMIN_COURTS_ENDPOINT + AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_SLUG
+        + AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_AREAS_OF_LAW + LOCAL_AUTHORITIES_PATH;
+    //changed local authority to existing one on database.
+    private static final String TEST = "Barnet Borough Council";
+    private static final int TEST_ID = 397_243;
+
+
+    //"/{slug}/{areaOfLaw}/localAuthorities")
```

Comment:
@ashwini-mv We can remove this comment here

Reply:
removed comment